R/finalize_plot.R

```diff
@@ -159,26 +159,26 @@ finalize_plot <- function(plot = NULL,
   }
 
   # create list of plot constants, from globals unless overridden by user
-  plot_constants <- utils::modifyList(cmapplot_globals$plot_constants, overrides)
+  consts <- utils::modifyList(cmapplot_globals$consts, overrides)
 
   # convert function arguments (given in inches) to bigpoints
-  plot_constants <- append(
-    plot_constants,
+  consts <- append(
+    consts,
     list(
       height = convertUnit(unit(height, "in"), "bigpts", valueOnly = TRUE),
       width = convertUnit(unit(width, "in"), "bigpts", valueOnly = TRUE),
       title_width = convertUnit(unit(title_width, "in"), "bigpts", valueOnly = TRUE),
-      margin_v1_v2 = plot_constants$margin_v1 + plot_constants$margin_v2
+      margin_topline_tb = consts$margin_topline_t + consts$margin_title_t
```

Comment:
second item in this list should be _b, no?

Reply:
oh, wait, nevermind. I misread your new taxonomy. sorry!

Reply:
yes, good catch

Reply:
ah yes, right, that was actually what was intended. I misread it when relooking haha